.goreleaser.yml

```diff
@@ -12,7 +12,7 @@ builds:
     goarch:
       - amd64
 checksum:
-  name_template: '_GOOS_-checksums.txt'
+  name_template: '{{ .Runtime.Goos }}-{{ .Runtime.Goarch }}-checksums.txt'
```

Comment:
Prevents clashing when releasing macOS checksum files, as we now have two different architectures, and goreleaser complains that there's already a `darwin-checksum.txt` in place.